lhotse/cut/base.py

```diff
@@ -510,14 +510,16 @@ def trim_to_alignments(
         self,
         type: str,
         max_pause: Optional[Seconds] = None,
+        max_segment_duration: Optional[Seconds] = None,
         delimiter: str = " ",
         keep_all_channels: bool = False,
     ) -> "CutSet":  # noqa: F821
         """
         Splits the current :class:`.Cut` into its constituent alignment items (:class:`.AlignmentItem`).
         These cuts have identical start times and durations as the alignment item. Additionally,
         the `max_pause` option can be used to merge alignment items that are separated by a pause
-        shorter than `max_pause`.
+        shorter than `max_pause`. If `max_segment_duration` is specified, the merging will be
+        performed only if the resulting segment is shorter than `max_segment_duration`.
```

Comment:
This doc suggests to me that when the resulting segment is longer than `max_segment_duration`, the items would be left unmerged; that's not true, is it?